.rubocop.yml

```diff
@@ -43,7 +43,7 @@ Style/MultilineBlockChain:
   Enabled: false
 
 
```

Comment:
Could you please add the following cop config? We have this one disabled in the rest of our chef code base, as it adds little value and can be confusing if you are not aware of this syntax.

Style/HashSyntax:
  EnforcedShorthandSyntax: never

Reply:
Done